central/resourcecollection/datastore/datastore_impl.go

```diff
@@ -80,38 +101,56 @@ func (ds *datastoreImpl) initGraph() error {
 
 	// then add edges by batches
 	for i := 0; i < len(ids); i += initBatchSize {
-		var objs []*storage.ResourceCollection
+		var parents []*storage.ResourceCollection
 		if i+initBatchSize < len(ids) {
-			objs, _, err = ds.storage.GetMany(ctx, ids[i:i+initBatchSize])
+			parents, _, err = ds.storage.GetMany(ctx, ids[i:i+initBatchSize])
 		} else {
-			objs, _, err = ds.storage.GetMany(ctx, ids[i:])
+			parents, _, err = ds.storage.GetMany(ctx, ids[i:])
 		}
```

Comment:
initBatchSize is a value defined in the package, I don't think we want to reset this value? It would be a const if it was in c

Reply:
Why is it global?

Reply:
It's a configurable value defined at the top of the package. Is this not an idiomatic way of handling the idea of configurable values in go? We define a number of objects we want to handle as a batch size because of the concern that we might hit a point where querying all will consume too much memory. If in the future we decide we want to raise this number we only have to change the value in one place. If I move this value under a `const()` block would that make it better?

Reply:
Fair enough, although this var is meant only for one time use, that is during init, unlike `batchSize` constant in other places in code which are used across multiple ops. 

Defining `initBatchSize` communicates more clearly.

Reply:
I went ahead and made it `graphInitBatchSize` just for extra clarity